src/coreclr/src/vm/gctoclreventsink.cpp

```diff
@@ -25,6 +25,7 @@ void GCToCLREventSink::FireDynamicEvent(const char* eventName, void* payload, ui
 
 void GCToCLREventSink::FireGCStart_V2(uint32_t count, uint32_t depth, uint32_t reason, uint32_t type)
 {
+#ifndef TARGET_ANDROID
```

Comment:
Why is this ifdefed out?

Reply:
Changed to `#ifdef FEATURE_EVENT_TRACE`, as we have the feature disabled for Android.